grass7/imagery/i.sentinel/i.sentinel.import/i.sentinel.import.py

```diff
@@ -504,8 +537,12 @@ def main():
                          "when -{} flag given").format('p'))
         importer.print_products()
         return 0
-    
+
     importer.import_products(flags['r'], flags['l'], flags['o'])
+
+    if options['footprints']:
```

Comment:
Could you include set_footprint() into import_products(), setting and limiting the temporary region to the footprint could speedup the import, esp. when done with projection (r.import). So the procedure would be:
1. set temp region to footprint (if provided and options["extent"] != "region")
2. import
3. Clip to footprint if provided or round if import was done this r.import (r.proj) (see: #127)

Reply:
to 1.) We might want to allow both extent=region AND footprints, e.g. if we are just interested in a small area (that might be covering an edge of a footprint)

Reply:
set_footprint() is now moved into import_products()